dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -321,5 +323,14 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
   def getNumericType(): TensorDataType = {
     ev.getType()
   }
+
+  /**
+   * module predict
+   * @param dataset dataset for prediction
+   * @param batchSize batch size per partition
+   */
+  def predict[D](dataset: RDD[D], batchSize: Int): RDD[Tensor[T]] = {
```

Comment:
ok, i will refactor the interface. the input dataset is RDD[Sample], and the output result also RDD[Sample], same to PychonBigDL.modelPredictRDD()